packages/api-gateway/config/cors.js

```diff
@@ -0,0 +1,5 @@
+const prefix = require('./environment');
+
+module.exports = {
+  domain: process.env[`${prefix}CORS_DOMAIN`],
```

Comment:
Do you talk with cohort about third services needed? If that's the case, this value should be an array. In order to accomplish that we need to separate values with commas and use split function.
[Here](https://stackoverflow.com/questions/63846589/how-to-define-array-object-in-env-file) more information about creating an array using env vars